infrastructure/EmailSender.py

```diff
@@ -3,8 +3,11 @@
 from email.mime.text import MIMEText
 
 from domain.models import MailBody
-
-from local_auth import login, password
+import os
+try:
+    from local_auth import login, password
```

Comment:
Zrobiłbym na odwrót, że domyślnie patrzy czy w zmiennych środowiskowych są takie wartości, a jak nie ma to próbuje pobrać from local_auth, ale to w sumie detale XD

Reply:
Zanczy wiesz, tu mi to python ogarnia, bo próbuje zaimportować, a jak nie może, to rzuca wyjątkiem. Tak musiałbym sam jakoś rzucać wyjątkami i potem patrzeć, czy jest local_auth, a jak nie ma, to dopiero wtedy rzucać wyjątkiem, to nie bardzo jest to opłacalne XD ale jak mi każesz, to postaram się zrobić tak, jak mówisz XD

Reply:
Znaczy os.environ to słownik XDXD
```python3
def get_from_local_auth(key):
   from local_auth import login, password
   ...
login_key = "EMAIL_LOGIN"
login = environ[login_key] if login_key in environ else get_from_local_auth(login_key)
```
Coś tego typu by było XD

Reply:
Ale może być też twoje po prostu nie przepadam za rzucaniem wyjątków jeśli można tego uniknąć XDXD

Reply:
Ale nie wiem czy jest to clean codowe. Więc może być twoje. Ogólnie jak chcesz zostawić to mogę zmergować, bo jest good enough.

Reply:
Znaczy jak zmienisz na approve, to mogę zmergować XD